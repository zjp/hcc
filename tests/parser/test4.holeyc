void main() {
	int age;
	TOCONSOLE "enter your age: ";
	FROMCONSOLE age;
	TOCONSOLE "\nyour age is";
	TOCONSOLE age;
}
