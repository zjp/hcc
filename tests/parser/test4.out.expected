void main(){
	int age;
	TOONSOLE "enter your age: ";
	FROMCONSOLE age;
	TOCONSOLE "\nyour age is";
	TOCONSOLE age;
}
